pkg/image/resolve.go

```diff
@@ -313,6 +313,7 @@ func setRequirementImages(osType OSType, imagesSet map[string]map[string]bool) {
 	switch osType {
 	case Linux:
 		addSourceToImage(imagesSet, settings.ShellImage.Get(), coreLabel)
+		addSourceToImage(imagesSet, settings.MachineProvisionImage.Get(), coreLabel)
```

Comment:
Could also be system as I think that's what other RKE related images are under but this works too.